src/pages/index.js

```diff
@@ -0,0 +1,71 @@
+import Head from "next/head";
+import { useEffect, useState } from "react";
+import { useSession } from "@inrupt/solid-ui-react";
+import { getPersonName } from "../utils/participantsHelper";
+import { getRDFasJson } from "../utils/fetchHelper";
+
+export default function Home() {
+  const { session, sessionRequestInProgress } = useSession();
+  const [name, setName] = useState("");
+
+  useEffect(() => {
+    const webID = session.info.webId;
+    if (webID !== undefined) {
+      const frame = {
+        "@context": {
+          "@vocab": "http://xmlns.com/foaf/0.1/",
+          knows: "https://data.knows.idlab.ugent.be/person/office/#",
+          schema: "http://schema.org/",
+        },
+        "@id": webID,
+      };
+
+      (async () => {
+        const data = await getRDFasJson(webID, frame, fetch);
+        setName(getPersonName(data) || webID);
+      })();
+    }
+  }, [session.info.webId]);
+
+  if (sessionRequestInProgress) {
+    console.log(sessionRequestInProgress);
+    return <p>Loading...</p>;
+  }
+
+  return (
+    <div>
+      <Head>
+        <title>Solid Calendar</title>
+        <meta name="description" content="Calendar using solid protocol" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main>
+        {session.info.isLoggedIn ? (
+          <>
+            <p>Welcome {name}!</p>
+            <h3>Availability Page</h3>
+            <p>View or edit your schedules on this page</p>
+            <h3>Contacts Page</h3>
+            <p>View your contacts and arrange meetings</p>
+            <h3>Meetings Page</h3>
+            <p>
+              View or edit your booked meetings that have been booked through
+              solid calendar
+            </p>
+          </>
+        ) : (
+          <>
+            <h3>What is this app?</h3>
```

Comment:
The app should immediately show the functionality. There is no need to explain the app when opening it. If that is the case then there are problems with the user experience or user interface.

Reply:
If the user has to login first anyways, what would you show when the user hasn't logged in yet?

Reply:
In that case you show the login options directly. Now people first have to click login at the top and then do the login stuff. This initial click can be avoided by showing the login options directly when launching the app.

You can still have a description of what the app does, but I would remove the header "What is this app?" and rewrite the paragraph to focus on user experience and not what it is from a development standpoint. The fact that it is an alternative to Doodle is kinda irrelevant for the user. What it will actually provide them is what matters.